CHANGELOG.asciidoc

```diff
@@ -127,7 +127,7 @@ filebeat.inputs:
 - Add AWS Cloudwatch capability to retrieve tags from AWS/ApiGateway resources. {pull}40755[40755]
 - Add new metrics for the vSphere Virtualmachine metricset. {pull}40485[40485]
 - Add `metrics_count` to Prometheus module if `metrics_count: true` is set. {pull}40411[40411]
-
+- Add `use_performance_counters` to use performance counters to collect CPU related metrics on Windows. {pull}41965[41965]
```

Comment:
Don't edit the `CHANGELOG.asciidoc`, instead edit the `CHANGELOG.next.asciidoc`.

`CHANGELOG.asciidoc` gets updated during the release process.

Reply:
Oh, sorry. I'll update `CHANGELOG.asciidoc`